public-ui/libproject/devstart.rb

```diff
@@ -0,0 +1,248 @@
+# UI project management commands and command-line flag definitions.
+
```

Comment:
This is nearly a carbon-copy of ui/libproject/devstart.rb, right? 

It would probably make sense to have a common-ui dir (similar to common-api) where Ruby or Typescript or whatever that we want to share between ui and public-ui can live. Then the files in ui and public-ui can just import it.

Reply:
Ok.  I'll give factoring a try.

Reply:
Factored into common-ui/commonu-ui-devstart.rb  .. I deployed public-ui to test. 
  https://public-ui-dot-all-of-us-workbench-test.appspot.com